### PR TITLE
[chore] add example for using recombine in container operator

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

File: pkg/stanza/docs/operators/container.md [hunk 1]
```diff
@@ -251,4 +251,72 @@ Configuration:
 
 </td>
 </tr>
-</table>
\ No newline at end of file
+</table>
+
+#### Parse multiline logs and recombine into a single one
+
+If you are using the Docker format (or log tag indicators are not working),
```
[the remark this answers is not in the document]
Comment (zeitlinger) [reply pkg/stanza/docs/operators/container.md:258]: good point - changed to docker

File: pkg/stanza/docs/operators/container.md [hunk 2]
```diff
@@ -251,4 +251,71 @@ Configuration:
...
+
+<table>
+<tr><td> Input body </td> <td> Output body</td></tr>
+<tr>
+<td>
+
+```json lines
+{
+  "timestamp": "",
+  "body": "{\"log\":\"2024-07-03T13:50:49.526Z  WARN 1 --- [http-nio-8080-exec-6] c.m.antifraud.FraudDetectionController   : checkOrder\",\"stream\":\"stdout\",\"time\":\"2029-03-30T08:31:20.545192187Z\"}",
```
Comment (ChrsMark) [inline pkg/stanza/docs/operators/container.md:285]: nit: the year in the `time` field is not the same as in the parsed `timestamp`:
`2029-03-30T08:31:20.545192187Z` VS `2024-03-30 08:31:20.545192187 +0000 UTC` (2029 vs 2024).

Same for the next log line.

**Review by ChrsMark — APPROVED** [review]
LGTM, thank's for adding this!

@djaglowski please take a look when you get the chance.

File: pkg/stanza/docs/operators/container.md (hunk 2 above)
Comment (zeitlinger) [reply pkg/stanza/docs/operators/container.md]: good catch

**Review by djaglowski — APPROVED** [review]
Nice, thanks @zeitlinger

**Pull request merged**